.env.cypress

```diff
@@ -0,0 +1,4 @@
+dhis2_base_url=https://debug.dhis2.org/ca-test-2.34
+dhis2_username=admin
+dhis2_password=district
```

Comment:
Is there a need for those not to be capital?
like `DHIS2_BASE_URL` for instance??

Reply:
I agree, they should be capitalized. Changing 👍

Reply:
Had to roll back this change. For some reason the login command won't resolve the environment variables when they are capital letters (at least that is my working theory). Will delve into this later at some point.